help/en/html/tools/logixng/reference/chapter6.shtml

```diff
@@ -311,6 +311,17 @@
         </li>
       </ul>
 
+      <p>The <strong><em>Type</em></strong> indicates the how the comparison process works.</p>
+
+      <ul>
+        <li><strong>Number or string</strong> &mdash; The comparison is based on the characteristics
+        of the objects.</li>
+        <li><strong>String</strong> &mdash; The comparison is based on the string values of the
+        objects.</li>
+        <li><strong>Number</strong> &mdash; The comparison is based on the numeric values of the
+        objects.  This can result in an exception if both values are not numbers.</li>
```

Comment:
Maybe this should be:
`This can result in an exception if any of the values is not a number.`

The point is that both numbers needs to be a number.

Reply:
Or maybe:
`This will result in an exception if any of the values is not a number.`